include/net/gptp.h

```diff
@@ -278,6 +296,14 @@ void gptp_foreach_port(gptp_port_cb_t cb, void *user_data);
  */
 struct gptp_domain *gptp_get_domain(void);
 
+/**
+ * @brief This interface is used by the ClockSource entity to provide time to
+ *        the ClockMaster entity of a time-aware system.
+ *
+ * @param arg Current state and parameters of the ClockSource entity.
+ */
+void gptp_clock_source_time_invoke(struct gptp_clk_src_time_invoke_params arg);
```

Comment:
Shouldn't the parameter be a pointer here?

Reply:
Sure, it should, will change that, thanks!